plonky2/src/gates/poseidon2_babybear.rs

```diff
@@ -31,8 +31,8 @@ use crate::plonk::config::AlgebraicHasher;
 use crate::plonk::vars::{EvaluationTargets, EvaluationVars, EvaluationVarsBase};
 use crate::util::serialization::{Buffer, IoResult, Read, Write};
 
-const USE_INTERNAL_PERMUTATION_GATE: bool = true;
-const USE_EXTERNAL_PERMUTATION_GATE: bool = true;
+const USE_INTERNAL_PERMUTATION_GATE: bool = false;
+const USE_EXTERNAL_PERMUTATION_GATE: bool = false;
```

Comment:
Can you please add a comment describing the reason why these two flags were introduced and why they are now disabled?

Thanks!